docs/appendices/release-notes/6.0.0.rst

```diff
@@ -266,6 +266,10 @@ Data Types
 Scalar and Aggregation Functions
 --------------------------------
 
+- Added support for :ref:`NUMERIC type<type-numeric>` to the following
```

Comment:
Shouldn't we also add a breaking changes entry related to that? AFAIK, numeric arguments worked due to coercion, but yield different return types, right?

Reply:
don't know, in `5.10.0` we didn't do: https://github.com/crate/crate/pull/16825